api/envoy/extensions/compression/gzip/decompressor/v3/gzip.proto

```diff
@@ -20,7 +20,8 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 message Gzip {
   // Value from 9 to 15 that represents the base two logarithmic of the decompressor's window size.
   // The decompression window size needs to be equal or larger than the compression window size.
-  // The default is 12 to match the default in the
+  // The default window size is 15.
+  // This is so that the decompressor can decompress a response compressed by a compressor with any compression window size.
   // :ref:`gzip compressor <envoy_api_field_extensions.compression.gzip.compressor.v3.Gzip.window_bits>`.
```

Comment:
This reference to the compressor becomes unneeded with this patch. Should be dropped.

Reply:
On it.